freetextresponse/freetextresponse.py

```diff
@@ -322,13 +362,12 @@ def _get_indicator_class(self):
         """
         Returns the class of the correctness indicator element
         """
-        result = ''
-        if self.count_attempts == 0:
-            result = 'unanswered'
-        elif self._determine_credit() == Credit.zero:
-            result = 'incorrect'
-        else:
-            result = 'correct'
+        result = 'unanswered'
+        if self.display_correctness and self._word_count_valid():
```

Comment:
@stvstnfrd  You are reading the condition correctly, but responses that don't meet the minimum word count are invalid and therefore not accepted as a submission.  They are not scored and count_attempts is not incremented.  This was original logic in our advanced xblock. 

I was curious so I just checked it out in studio.  Basic text responses allow users to submit empty responses for credit(which will always be incorrect) because instructors can't set the correct answer to an empty string.  I got an error in studio when I tried this.  

We could mimic this behavior, but then we'll need to display the min word count on the xblock prior to submission.  Currently we display the word requirements after submission.